packages/core/src/external/carequality/ihe-gateway-v2/saml/saml-client.ts

```diff
@@ -23,6 +24,20 @@ const { log } = out("Saml Client");
 const httpTimeoutPatientDiscovery = dayjs.duration({ seconds: 60 });
 const httpTimeoutDocumentQuery = dayjs.duration({ seconds: 120 });
 const httpTimeoutDocumentRetrieve = dayjs.duration({ seconds: 120 });
+
+const httpCodesToRetry: NetworkError[] = [
+  "ECONNREFUSED",
+  "ECONNRESET",
+  "ETIMEDOUT",
+  "ECONNABORTED",
+];
+
+const httpCodesToRetryPatientDiscovery: NetworkError[] = [...httpCodesToRetry];
+
+const httpCodesToRetryDocumentQuery: NetworkError[] = [...httpCodesToRetry, "ERR_BAD_RESPONSE"];
+
+const httpCodesToRetryDocumentRetrieve: NetworkError[] = [...httpCodesToRetry, "ERR_BAD_RESPONSE"];
```

Comment:
Just to confirm, this is for cases diff than that ones where we get 502/504, right? Just b/c on those we moved from `ERR_BAD_RESPONSE` to http status.